docs/specification/appendix.md

```diff
@@ -0,0 +1,111 @@
+# MCAP File Format Specification Appendix
+
+This document describes well known values for MCAP files.
+
+## Well-known compression formats
+
+The Chunk `compression` field may contain the following options:
+
+- [lz4](<https://en.wikipedia.org/wiki/lz4_(compression_algorithm)>): an algorithm that prioritizes compression/decompression speed over compression ratio.
+- [zstd](https://en.wikipedia.org/wiki/zstandard): an algorithm that prioritizes compression ratio over compression/decompression speed.
```

Comment:
I realize this text has probably read this way for a while, but I think a better description for zstd is “an algorithm with a tunable compression ratio/speed tradeoff”.